tools/docker/tests-runner/Dockerfile

```diff
@@ -6,7 +6,6 @@ RUN apk add --update --no-cache \
     python3 \
     grep \
     coreutils \
-    tcpdump \
```

Comment:
But why do we keep `tshark` and not replace it completely with `wireshark-common`? This also makes me notice that the PR name should be changed - "use dumpcap insteda of tcpdump".

Reply:
Because we still use tshark to parse the pcap file.